guide/16-geoenrichment-guides/part4_enrich_collections_and_sedf.ipynb

```diff
@@ -0,0 +1,2937 @@
+{
```

Comment:
<p>Looking at <code>fieldCategory</code> is a great way to clearly understand what the data collection is about. <code>fieldCategory</code> combines <code>vintage</code>, <code>datacollectionID</code> columns <s>and clearly clarifies</s> <strong>along with the</strong> year and data collection.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>